README.md

```diff
@@ -1,35 +1,116 @@
+    ```yaml
+    lightweight: true
+    plugin_alias: th2_service
+    name: th2_service
+    build_number: 0
+    revision: 0
+    git_hash: 0
+    branch: fake
+    version: 3.2.0.0
+    core_version: 3.2.0
+    ```
+4. Create you own image based on the current one and put all files to the correct places in the base image:
+    + Create the following directory - **${workspace}/plugins/th2_service**.
+    _**${workspace}**_ - it is a folder from the "Connect" configuration.
+    If you use the _plugin_alias_ and _name_ different from _th2_service_ in the VERSION file correct the _th2_service_ folder name according to value you use.
+    Let's name that directory as **PLUGIN_DIRECTORY** for simplicity. This name will be used in future steps.
+    + Artifact with the service(s) implementation(s) and all its dependencies should be put to the following directory - **${PLUGIN_DIRECTORY}/libs**.
+    + The configuration file created on the step 2 should be put to the following directory - **${PLUGIN_DIRECTORY}/cfg**.
+    + The _VERSION_ file created on step 3 should be put to the following directory - **${PLUGIN_DIRECTORY}/**.
+
+## Pins
+
+Connect has 4 types of pins for interacting with th2 components. Messages that were received from / sent to the target system will be sent to the following queues:
+
+- incoming parsed messages // it will be removed in future
+- outgoing parsed messages // it will be removed in future
 - incoming raw messages
 - outgoing raw messages
 
-To send messages, a separate queue is used. The Connectivity component subscribes to that queue at the start and waits for the messages - to be sent to the target system - to come from the queue.
-Also, this component is responsible for maintaining connections and sessions in the cases where this is provided by the communication protocol. Here you can automatically send heartbeat messages, send a logon/logout, requests to retransmit messages in the event of a gap, etc.
-
-
-## Environment variables
-
-- RABBITMQ_PASS=some_pass
-- RABBITMQ_HOST=some-host-name-or-ip
-- RABBITMQ_PORT=7777
-- RABBITMQ_VHOST=someVhost
-- RABBITMQ_USER=some_user
-- GRPC_PORT=7878
-- DISABLE_GRPC=false
-- SESSION_ALIAS=fix_client
-- TH2_EVENT_STORAGE_GRPC_HOST=some-host-name-or-ip
-- TH2_EVENT_STORAGE_GRPC_PORT=some-port
-- IN_QUEUE_NAME=in_queue
-- IN_RAW_QUEUE_NAME=in_raw_queue
-- OUT_QUEUE_NAME=out_queue
-- OUT_RAW_QUEUE_NAME=out_raw_queue
-- TO_SEND_QUEUE_NAME=to_send_queue
-- TO_SEND_RAW_QUEUE_NAME=to_send_queue
\ No newline at end of file
+The "Connect" component uses a separate queue to send messages. The component subscribes to that pin at the start and waits for the messages.
+The messages received from that pin will be sent to the target system.
+Also, this component is responsible for maintaining connections and sessions in the cases where this is provided by the communication protocol.
```

Comment:
**Besides**, this component is responsible for maintaining connections and sessions **,** **in the cases provided** by the communication protocol.